### ISSUE TITLE
The Module GPT-4o is not really GPT-4o

### ISSUE THREAD

**Issue by nietzschescodes** [josStorer/chatGPTBox#797]
I mean ChatGPT's GPT-4o browses the internet to get up-to-date info and the module "GPT-4o" doesn't browse the internet.

**Comment by Mohamed3nan** [comment]
Yes, I can confirm that there is an issue when asking to browse; it works as expected if you send the same prompt via the official ChatGPT website. However, this issue does not occur when changing the 'Custom ChatGPT Web API URL' to any working mirror, which indicates that the problem is specific to "chatgpt.com" @josStorer

**Comment by josStorer** [comment]
There's a control parameter for internet browsing that ChatGPTBox isn't passing through. I think this could be offered as an option.

**Comment by DRYN07** [comment]
Any news about this? None of the models can browse internet in ChatGPTbox (Gemini Web, ChatGPT API, Claude Web)
